src/header-navigation/examples.js

```diff
@@ -0,0 +1,265 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+/* eslint-disable react/display-name*/
+
+import * as React from 'react';
+import {Button} from '../button';
+import {StatefulMenu as Menu} from '../menu';
+import {StatefulSelect as Search} from '../select';
+import {styled} from '../styles';
+import COLORS from '../select/examples-colors';
+
+import {HeaderNavigation, ALIGN, NavigationItem, NavigationList} from './index';
+import tests from './examples-list';
+import {TYPE} from '../select';
+
+const Link = styled('a', props => ({
+  ':hover': {
+    color: props.$theme.colors.primary400,
+  },
+}));
+
+let ITEMS = [];
+for (let i = 0; i < 100; i++) {
+  ITEMS.push({label: `Item ${i}`});
+}
+class MenuContainer extends React.Component<
+  {isOpen: boolean, align: string},
+  {isOpen: boolean},
+> {
+  static defaultProps = {
+    isOpen: false,
+    align: '',
+  };
+  state = {
+    isOpen: this.props.isOpen,
+  };
+  render() {
+    return (
+      <React.Fragment>
+        <span
+          onClick={() =>
+            this.setState({
+              isOpen: !this.state.isOpen,
+            })
+          }
+        >
+          <svg
```

Comment:
can we move the svg to a file and import it, just to make this example more readable?

Reply:
ok